ods_ci/tests/Resources/CLI/MustGather/get-must-gather-logs.sh

```diff
@@ -3,6 +3,8 @@
 # process buffer and prevents the script from running further.
 oc adm must-gather --image=quay.io/modh/must-gather@sha256:1bd8735d715b624c1eaf484454b0d6d400a334d8cbba47f99883626f36e96657 &> must-gather-results.txt
 
+cat must-gather-results.txt
+
```

Comment:
Do we want to print this every time or was this used just for debugging this?

Reply:
good catch, I think we can remove it, because the output can be so big depending on the cluster